src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -45,17 +45,44 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   if (!highlighted) {
     str = markdownIt.utils.escapeHtml(str);
   }
+  let highlightLines = token.attrGet('highlight-lines')
+  let lineNumbers = []
+  if (highlightLines) {
+    lineNumbers = highlightLines.split(',').map(v => v.split('-').map(v => parseInt(v, 10))) 
+  }
+
   const lines = str.split('\n');
   lines.pop(); // last line is always a single '\n' newline, so we remove it
-  str =  lines.map(line => `<span>${line || '&#x200B;'}</span>`).join('');
+  str =  lines.map((line, index) => {
+    const currentLineNumber = index + 1
+    const inRange = lineNumbers.some(([start, end]) =>{
+      if (start && end) {
+        return currentLineNumber >= start && currentLineNumber <= end
+      }
+      return currentLineNumber === start
+    })
+    if (inRange) {
+      return `<span class="highlighted">${line || '&#x200B;'}</span>`;
```

Comment:
I think we can abstract the section `${line || '&#x200B;'}` into a separate variable, to make it clearer that this is identical across both cases.

Reply:
Addressed.